src/aruco_pose/README.md

```diff
@@ -45,4 +45,33 @@ In the current implementation, this topic's publish rate is defined in the Azure
 ## Median estimation
 Median pose estimation is a multi-value estimation, where it estimates the median of the 6 DoF pose returned by the camera. 
 
-For each DoF, it maintains a moving window of size 10 (can be changed via the parameter `number_of_observations` ) to calculate the median. When the detection rate is 5Hz and upon moving the sample or the detected object to a new pose, it takes a minimum of 1.2 seconds to return the correct pose for the updated pose. As a good practice, it is better to wait for the whole 2 seconds. 
\ No newline at end of file
+For each DoF, it maintains a moving window of size 10 (can be changed via the parameter `number_of_observations` ) to calculate the median. When the detection rate is 5Hz and upon moving the sample or the detected object to a new pose, it takes a minimum of 1.2 seconds to return the correct pose for the updated pose. As a good practice, it is better to wait for the whole 2 seconds. 
+
+# Connection to the Redis DB
+
+Redis is used to store the aruco tag ID and real-world sample information. At present, each entry has four properties. They are:
+- id  -> tag ID of the printed tag from AruCo marker library
+- family  -> dictionary of tag used [DICT_APRILTAG_36h11, DICT_6X6_250, etc... ]
+- size  -> physical size of the marker
+- sample_names  -> unique name to identify the sample used with the tag
+
+Run the Redis container with the following command (make sure to have the correct binding for the local storage):
+```bash 
+podman run --name redis-container --network host -d -v /home/wfernando1/Documents/Environments/Workspaces/redis/data/:/data -p 6379:6379 redis
+```
+
+## Insert new entires
+
+Run the following if the command line interface is needed:
+
+```bash
+podman exec -it redis-container redis-cli
+```
+
+New records can be inserted into the redis server with the following command. Note the below example assumes that each entry has four attributes: id, family, size, and sample_name.
+
+```bash
+HSET tag:1 id 0 family "DICT_APRILTAG_36h11" size 0.02665 sample_name sample_1
+```
```

Comment:
Can we also add a python example?

Reply:
Python example is added. https://github.com/maffettone/erobs/blob/281d3dfdb27f2387ff8b83695a9f3b0b94f6624b/src/aruco_pose/README.md?plain=1#L85-L90